package.json

```diff
@@ -3,8 +3,9 @@
   "version": "0.0.1",
   "description": "Cracking the Coding Interview - Answers in Javascript",
   "scripts": {},
```

Comment:
There's also the original scripts for linting and unit-testing. Only the "test" command should remain now. And that should call eslint on all the source files.